packages/byods/src/types.ts

```diff
@@ -0,0 +1,166 @@
+/**
+ * Represents the configuration options for the SDK.
+ * @interface SDKConfig
+ * @property {string} clientId - The client ID.
+ * @property {string} clientSecret - The client secret.
+ * @property {string} [baseUrl='https://webexapis.com/v1'] - The base URL for the API requests.
```

Comment:
This will be removed as we discussed in our review today?

Reply:
Updated the code to use `process.env.BYODS_ENVIRONMENT`